src/display/canvas.js

```diff
@@ -856,7 +856,10 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
 
       if (this.transparentCanvas) {
         this.ctx = this.compositeCtx;
+        this.ctx.save();
```

Comment:
Just tried and it doesn't work. Because the initial transformation for the canvas is not identity so restore it will not undo the transformation.